display_list/geometry/dl_path.h

```diff
@@ -35,12 +37,21 @@ class DlPath {
   DlRect GetBounds() const;
 
   bool operator==(const DlPath& other) const;
+  bool operator!=(const DlPath& other) const { return !(*this == other); }
 
   bool IsConverted() const;
+  bool IsVolatile() const;
 
  private:
-  const SkPath sk_path_;
-  mutable impeller::Path path_;
+  struct Data {
+    explicit Data(const SkPath& path) : sk_path(path) {}
+
+    SkPath sk_path;
+    impeller::Path path;
+    uint32_t use_count = 0u;
+  };
+
+  std::shared_ptr<Data> data_;
```

Comment:
🙈

Reply:
This shared_ptr enables global conversion and volatile tracking across all instances. It will collapse with another shared_ptr when we stop serving two backends and settle on a single impl.